documentation/Glossary.md

```diff
@@ -32,7 +32,7 @@
 
 ## AboutInfo
 
-Exposes information about a plugin, such as its authors, copyright, and license. See [Adding About Information](./Using-the-SDK/Advanced/Adding-About-Information.md).
+Exposes information about a plugin, such as its authors, copyright, and license.
```

Comment:
We can update this link to the section you added. If you specify `#heading` at the end of the URL it'll link to a specific section

Reply:
Ah, that's useful. Thanks.